File: pset6/mario_more.py
from cs50 import get_int

# request and input validation
while True:
    h = get_int("Height: ")
    if h > 0 and h < 9:
        break

# drawing a pyramid
for i in range(1, h + 1):
    print(" " * (h - i), end="")
    print("#" * i, end="")
    print("  ", end="")
    print("#" * i)

File: pset1/mario_more.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h;
    h = get_int("Height: ");
    while (h < 1 || h > 8)
    {
        h = get_int("Height: ");
    }

    for (int i = 1; i < h + 1; i++)
    {
        for (int x = 1; x <= h - i; x++)
        {
            printf(" ");
        }
        for (int y = 1; y <= i; y++)
        {
            printf("#");
        }
        printf("  ");
        for (int y = 1; y <= i; y++)
        {
            printf("#");
        }
        printf("\n");
    }
}
